scripts/upgrade

```diff
@@ -6,8 +6,8 @@
 # IMPORT GENERIC HELPERS
 #=================================================
 
-source _common.sh
 source /usr/share/yunohost/helpers
+source _common.sh
```

Comment:
Seriously !?

I mean, dude you're supposed to be in the Apps group !

At least just say that you don't know or you don't want to do it, we'll do it. But come on dude...